### PR TITLE
Setup CI for Knative Serving v0.19.0

### PR DESCRIPTION
**Pull request by nak3**
As per title this patch sets up for serving v0.19.0.

/cc @markusthoemmes @mgencur

### CODE REVIEW

**Comment by markusthoemmes** [comment]
One more invalid image?

**Comment by nak3** [comment]
/retest
Hmm... servingcontainer was not changed and v0.18 can build it. Let's see one more test.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

Your wish is my command!

File: ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.19.0__4.6.yaml [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.1"
+  namespace: ocp
+test_binary_build_commands: make test-install
+tests:
+- as: e2e-aws
+  commands: make test-e2e
+  openshift_installer_src:
```
Comment (alvaroaleman) [inline ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.19.0__4.6.yaml:253]: @nak3 please do not add new template-based tests and use multistage instead, see https://docs.ci.openshift.org/docs/how-tos/migrating-template-jobs-to-multistage/#openshift_installer_src

/hold

Comment (markusthoemmes) [reply ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.19.0__4.6.yaml:253]: Ehr, good call :thinking: Why deviate from master and 0.18 here @nak3 ?

Comment (nak3) [reply ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.19.0__4.6.yaml:253]: Thank you. I willl fix it.

> Why deviate from master and 0.18 here @nak3 ?

I generated the file by `make BRANCH=release-v0.19.0 generate-ci-config` in openshift/knative-serving. I will update the script - https://github.com/openshift/knative-serving/blob/master/openshift/ci-operator/generate-ci-config.sh#L27 :sweat_smile:

Comment (markusthoemmes) [reply ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.19.0__4.6.yaml:253]: Oh yeah, it's horribly out of date :joy:.

Just copy master and adjust as necessary.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**